Release/QuickChat/docs/README.md

```diff
@@ -12,14 +12,15 @@ In this documentation, you will learn how to add a QuickChat module in your exis
   
 - Select the quick chat module in the list of module on the left tap of the module importer
 - Click Import  to import the QuickChat Module. 
-- If you did not set up the multiplay feature, make sure that while importing the module you also import the `Zepeto Multiplay Component` and the `World.multiplay` (note the QuickChat feature works only in a multiplay world). In case you already have a multiplay or you are planning to add one on your own, feel free to uncheck the `Zepeto Multiplay Component` and the `World.multiplay` before clicking the import button at the bottom.
+- If you did not set up the multiplay feature, make sure to set up the multiplay component first. In case you already have a multiplay or you are planning to add one on your own, feel free to uncheck the `Zepeto Multiplay Component` and the `World.multiplay` before clicking the import button at the bottom.
```

Comment:
```suggestion
- If you did not set up the multiplay feature, make sure to set up the multiplay component first.
```

Since we're not going to include `World.Multiplay` and the multiplay component in the package, the second sentence here is invalid.